src/storage/mutate/DeleteEdgesProcessor.cpp

```diff
@@ -124,6 +124,12 @@ DeleteEdgesProcessor::deleteEdges(PartitionID partId,
         }
 
         if (iter->valid()) {
+            // TODO(liuyu): known issue
+            //      if lock deleted, but out-edge not delted, there may be in-consist
```

Comment:
If the in-edge is deleted, the out-edge is not deleted?
Or the out-edge is deleted, but the in-edge is not deleted?

so delete also support toss

Reply:
agree, will add this later in another pr